frontend/pages/ManageControlsPage/Secrets/Secrets.tsx

```diff
@@ -277,7 +234,7 @@ const Secrets = () => {
         <DeleteSecretModal
           secret={secretToDelete}
           onExit={() => setShowDeleteModal(false)}
-          reloadList={reloadList}
+          onDeleteSecret={onDeleteSecret}
```

Comment:
Technically the lowest-touch change would have been to keep `reloadList` and pass it `refetch` instead, but seemed better to make "delete" and "save" operate in the same way.